docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,575 @@
+        "lpar2": {
+            "properties": {
+                "host": "example2.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 1443
+                    }
+                }
+            }
+        },
+        "my_base": {
+            "type": "base",
+            "properties": {
+                "rejectUnauthorized": true
+            }
+        }
+    },
+    "defaults": {
+        "zosmf": "lpar1.zosmf",
+        "base": "my_base"
+    },
+    "plugins": [],
+    "secure": [
+        "profiles.my_base.properties.password",
+        "profiles.my_base.properties.user"
+    ]
+}
+```
+
+### Tips for using the base array
+
+The base array is a useful tool for sharing option values between services. You might define options to the base array in the following situations:
+- You have multiple services that share the same username, password, or other value.
+- You want to store a web token to access all services through Zowe API Mediation Layer.
+- You want to trust a known self-signed certificate, or your site does not have server certificates configured. You can define `reject-unauthorized` in the base array with a value of  `false` to apply to all services. Understand the security implications of accepting self-signed certificates at your site before you use this method.
+
+## Sharing global configuration
+
+You might want to share configuration in the following scenarios:
+- Share global config with developers so that they can begin working with a defined set of mainframe services. The recipient of the file manually places it in their local `~/.zowe` folder before issuing CLI commands.
+- Add global config to your project directory in an SCM tool such as GitHub. This lets other developers pull the project to their local machine and make use of the defined configuration. Zowe CLI commands that you issue from within the project directory automatically use the project's config scheme.
+- Enable test automation and CI/CD, letting your pipelines make use of the project configuration.
+
+## Example configurations
+
+<!-- TODO
+Shall we provide a handful of examples here of different use cases and the .json for each?
+
+Mike B - "I'd cover at least two. One for accessing multiple services directly on multiple LPARs that share the same username/password. One for accessing multiple services via the API ML (where MFA/SSO is achievable via token-based auth).
+
+Another example could be accessing multiple services directly on LPAR1 and LPAR2 where username/password varies between LPAR1 & LPAR2 services.
+
+Yet another example could be a situation where API ML is leveraged to access production services but services on a dev-test environment can be accessed directly."
+
+t1m0thyj - Added JSON examples below
+-->
+```json
+// accessing multiple services directly on multiple LPARs that share the same username/password
```

Comment:
For this and subsequent JSON examples, perhaps the comment at top of JSON should be moved out and converted to 1 or 2 sentences describing the example that follows?